certified-connectors/Cognizant Automation Center/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1105 @@
+        },
+        "summary": "Get Customer information based on API token",
+        "description": "Get Customer information based on API token",
+        "operationId": "FetchCustomerDetails",
+        "deprecated": false,
+        "x-ms-visibility": "internal",
+        "x-ms-api-annotation": {
+          "status": "production",
+          "revision": 1
+        },
+        "parameters": [
+          {
+            "name": "ttype",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "default": "Id",
+            "x-ms-visibility": "internal",
+            "x-ms-summary": "set ttype to string"
+          }
+        ]
+      }
+    },
+    "/portal/ino/api/v3/collab/activity/0/logs": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "Success": {
+                  "type": "boolean",
+                  "description": "Success"
+                },
+                "Msg": {
+                  "type": "string",
+                  "description": "Msg"
+                },
+                "TransactionKey": {
+                  "type": "string",
+                  "description": "TransactionKey"
+                }
+              }
+            }
+          }
+        },
+        "operationId": "CreateaActivityLog",
+        "description": "Create Activity Log",
+        "summary": "Create Activity Log",
+		"deprecated": false,
+        "x-ms-api-annotation": {
+          "status": "production",
+          "revision": 1
+        },
+        "parameters": [
+          {
+            "name": "ttype",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "default": "NewRecord",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "ActivityId": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "ActivityId",
+                  "x-ms-summary": "ActivityId"
+ 
+                },
+                "Comment": {
+                  "type": "string",
+                  "description": "Comment",
+                  "x-ms-summary": "Comment",
+				  "x-ms-visibility": "advanced",
+                  "default": "Triggered by PowerAutomate. Run id: @{workflow().run.name} Workflow: @{concat('https://india.flow.microsoft.com/manage/environments/',workflow().tags.environmentName,'/flows/',workflow().name,'/runs/',workflow().run.name)}"
+                },
+                "CreatedById": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "CreatedById",
+				  "x-ms-summary": "CreatedById"

```

Comment:
@SelvaGanesh-Cognizant Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

Reply:
@sriyen-msft  Ident issue is addressed. Please review it.